### PR TITLE
tikv: add a new config item `background-error-recovery-window`

### PR DESCRIPTION
**Pull request by hunterlxt**
Signed-off-by: Xintao <hunterlxt@live.com>

### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)
Add a new tikv config item `background-error-recovery-window`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by tonyxuqqi — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -383,6 +383,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 默认值：12h
 + 最小值：0s
 
+### `background-error-recovery-window`
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:386]:
```suggestion
### `background-error-recovery-window` <span class="version-mark">从 v6.1.0 版本开始引入</span>
```

File: tikv-configuration-file.md [hunk 2]
```diff
@@ -383,6 +383,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 默认值：12h
 + 最小值：0s
 
+### `background-error-recovery-window` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
++ RocksDB 检测到可恢复的后台错误后的最长恢复时间。当检测到故障 SST 所属的 Peer 后，会通过心跳上报到 PD。PD 随后会下发调度操作移除该 Peer。最后故障 SST 文件将会被直接删除，随后调度恢复正常。
```
Comment (TomShawn) [inline tikv-configuration-file.md:388]: 「随后调度恢复正常」是说 TiKV 后台恢复正常吗？
```suggestion
+ RocksDB 检测到可恢复的后台错误后，所允许的最长恢复时间。如果后台 SST 文件出现故障，RocksDB 在检测到故障 SST 文件所属的 Peer 后，会通过心跳上报到 PD。PD 随后会进行调度操作移除该 Peer。最后故障 SST 文件将会被直接删除，随后调度恢复正常。
```

File: tikv-configuration-file.md [hunk 3]
```diff
@@ -383,6 +383,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
++ 当恢复操作完成之前，损坏的 SST 文件将一直存在。此时 RocksDB 可以继续写入新的内容，但读到损坏的数据范围时会返回错误。
```
Comment (TomShawn) [inline tikv-configuration-file.md:389]:
```suggestion
+ 在恢复操作完成之前，损坏的 SST 文件将一直存在。此时 RocksDB 可以继续写入新的内容，但读到损坏的数据范围时会返回错误。
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -383,6 +383,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
...
++ 当恢复操作未能在该时间窗口内完成，TiKV 会主动 panic。
```
Comment (TomShawn) [inline tikv-configuration-file.md:390]:
```suggestion
+ 如果恢复操作未能在该时间窗口内完成，TiKV 会主动 panic。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -383,6 +383,13 @@ TiKV 配置文件比命令行参数支持更多的选项。你可以在 [etc/con
 + 默认值：12h
 + 最小值：0s
 
+### `background-error-recovery-window` <span class="version-mark">从 v6.1.0 版本开始引入</span>
+
++ RocksDB 检测到可恢复的后台错误后，所允许的最长恢复时间。如果后台 SST 文件出现损坏，RocksDB 在检测到故障 SST 文件所属的 Peer 后，会通过心跳上报到 PD。PD 随后会进行调度操作移除该 Peer。最后故障 SST 文件将会被直接删除，随后调度恢复正常。
```
Comment (Oreoxmt) [inline tikv-configuration-file.md:388]:
```suggestion
+ RocksDB 检测到可恢复的后台错误后，所允许的最长恢复时间。如果后台 SST 文件出现损坏，RocksDB 在检测到故障 SST 文件所属的 Peer 后，会通过心跳上报到 PD。PD 随后会进行调度操作移除该 Peer。最后故障 SST 文件将会被直接删除，随后 TiKV 后台恢复正常。
```

**Review by TomShawn — APPROVED** [review]
LGTM

**Pull request merged**